### PR TITLE
GDB-7014 Fix where scrollbars appear in dialogs

### PR DESCRIPTION
**Pull request by yordanalexandrov**
- refactored modal dialog views to have the `modal-body` with overflow auto and set modal-content to have max-height 90vh
 - set dropdown menus to toggle visibility with display: none/block rather than visibility: visible/hidden

### CODE REVIEW

File: src/js/angular/autocomplete/templates/modal/add-label.html [hunk 1]
```diff
@@ -18,12 +19,12 @@ <h3 ng-if="!isNew" class="modal-title">{{'add.label.edit.autocomplete' | transla
...
-    <div class="modal-footer">
-        <button type="button" class="btn btn-secondary" ng-click="cancel()">{{'common.cancel.btn' | translate}}</button>
-        <button id="wb-autocomplete-savegraph-submit" class="btn btn-primary" ng-click="ok()" type="submit">{{'common.ok.btn' | translate}}</button>
-    </div>
-</form>
+    </form>
+</div>
+<div class="modal-footer">
+    <button type="button" class="btn btn-secondary" ng-click="cancel()">{{'common.cancel.btn' | translate}}</button>
+    <button id="wb-autocomplete-savegraph-submit" class="btn btn-primary" ng-click="ok()" type="submit">{{'common.ok.btn' | translate}}</button>
```
Comment (svilenvelikov) [inline src/js/angular/autocomplete/templates/modal/add-label.html:26]: With these changes, is this the default button now?

File: src/js/angular/graphexplore/templates/modal/save-graph.html [hunk 2]
```diff
@@ -26,13 +26,13 @@ <h3 class="modal-title">{{title}}</h3>
              style="margin-top: 5px;">{{'save.graph.name' | translate}} <strong>&#39;{{graph.name}}&#39;</strong> {{'save.graph.exists' | translate}}
         </div>
         <!-- <div ng-if="graphExists || overwrite"><label for="override-saved">Overwrite existing saved graph <input id="override-saved" type="checkbox" ng-model="overwrite"></label></div> -->
```
Comment (svilenvelikov) [inline src/js/angular/graphexplore/templates/modal/save-graph.html:28]: If you do some other changes in this MR, would you delete this pls?

File: src/js/angular/templates/modal/add-location.html [hunk 3]
```diff
@@ -62,17 +62,17 @@ <h3 class="modal-title">{{'attach.remote.gdb.instance' | translate}}</h3>
         </div>
         <div class="form-group row" ng-if="newLocation.authType === 'signature'">
             <div class="col-md-12">
-                {{'signature.auth.info' | translate}}<a href="{{docBase}}/access-control.html#gdb-authentication"
-                                                        rel="noopener" target="_blank">Access Control / GDB authentication</a>{{'signature.auth.info.last' | translate}}
-            </div>
+            {{'signature.auth.info' | translate}}<a href="{{docBase}}/access-control.html#gdb-authentication"
+                                                    rel="noopener" target="_blank">Access Control / GDB authentication</a>{{'signature.auth.info.last' | translate}}
```
Comment (svilenvelikov) [inline src/js/angular/templates/modal/add-location.html:66]: This seems to be left untranslated.

Comment (yordanalexandrov) [reply src/js/angular/templates/modal/add-location.html:66]: This is actually a title for sections in the documentation. Not sure if it should be translated

File: src/js/angular/autocomplete/templates/modal/add-label.html (hunk 1 above)
Comment (yordanalexandrov) [reply src/js/angular/autocomplete/templates/modal/add-label.html]: No.
I fixed by adding `id` attribute to form and `form` attribute to button with value set to form id.
Now when enter is pressed the form is submited

**Pull request merged**